qiskit/compiler/transpiler.py

```diff
@@ -260,6 +262,10 @@ def callback_func(**kwargs):
             the ``backend`` argument, but if you have manually constructed a
             :class:`~qiskit.transpiler.Target` object you can specify it manually here.
             This will override the target from ``backend``.
+        hls_config: An optional configuration class that will be passed directly
+            to the high-level synthesis transformation pass. This configuration
+            class allows to specify for various high-level objects the lists of
+            synthesis algorithms and their parameters.
```

Comment:
Can we add a link to the `HLSConfig` class to help users know e.g. how to construct an `HLSConfig`?

Reply:
Added the link.